http-clients/src/main/java/org/triplea/http/client/web/socket/messages/MessageTypeListenerBinding.java

```diff
@@ -0,0 +1,25 @@
+package org.triplea.http.client.web.socket.messages;
+
+import com.google.errorprone.annotations.Immutable;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import lombok.Value;
+
+@Value(staticConstructor = "of")
+@Immutable
+// The function passed as second param isn't
```

Comment:
IMO, perhaps not worth the comment and the warning suppression to mark immutable. Could be just as well to leave it as effectively immutable.